### ISSUE TITLE
please add a new feature to automatically change endpoint when getting errors

### ISSUE THREAD

**Issue by Nicolas47400** [shakhruz/angelfarmers-ui#35]
Hi, please add a new feature to automatically change endpoint when getting failed to fetch, failed to claim error and more errors like when you failed to repair or recharge energy, i saw that feature on another bot:
![2032b104-75d5-4ba4-adee-42985538d220](https://user-images.githubusercontent.com/104126938/164394889-db9d1df3-1337-4364-911f-fb04281f51a2.jpg)

**Comment by shakhruz** [comment]
We used to have that feature before, worked worse than manually selecting a node that is consistent and work most of time. Maybe that could be an option.

**Comment by Nicolas47400** [comment]
Yes as an option of course, also if we can chose what nodes can be automaticaly selected it would be a+ because there is a few that are garbage, i think it would work better.

Thank you.
